<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kvizEdit</class>
 <widget class="QDialog" name="kvizEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uredi kviz</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ipi.jpg</normaloff>../ipi.jpg</iconset>
  </property>
  <widget class="QTableView" name="tbl_kvizEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>257</y>
     <width>428</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_ID">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ID">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_na_pi">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_na_pi"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_pr_od">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_pr_od"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_editKviz">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Uredi kviz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_editPitanje">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Uredi pitanje</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_addKviz">
        <property name="text">
         <string>Dodaj kviz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_addPitanje">
        <property name="text">
         <string>Dodaj pitanje</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_nazad">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
